horreum-web/src/domain/tests/AllTests.tsx

```diff
@@ -394,22 +372,20 @@ export default function AllTests() {
     const teams = useSelector(teamsSelector)
     const isAuthenticated = useSelector(isAuthenticatedSelector)
     const [rolesFilter, setRolesFilter] = useState<Team>(ONLY_MY_OWN)
+
     const [loading, setLoading] = useState(false)
     const [limit, setLimit] = useState(20)
     const [page, setPage] = useState(1)
     const [direction] = useState<SortDirection>("Ascending")
     const pagination = useMemo(() => ({ page, limit, direction, folder }), [page, limit, direction, folder])
     const [count, setCount] = useState(0)
 
+    const [folders, setFolders] = useState<string[]>([])
+
     const loadTests = () => {
-        setLoading(true)
         fetchTestsSummariesByFolder(alerting, SortDirection.Ascending, pagination.folder, pagination.limit, pagination.page, rolesFilter.key, )
-            .then(summary => {setTests(summary.tests?.map(t => mapTestSummaryToTest(t)) || [])
-                if (summary.count) {
-                    setCount(summary.count)
-                }
-            })
-            .finally( () => setLoading(false))
```

Comment:
I see the `loading` management has been removed, this means that `const [loading, setLoading] = useState(false)` would be no longer needed as no more used anywhere here. Was that intentional?

Reply:
@lampajr is this still an question? the code is flagged as "outdated"

Reply:
yeah it is still valid question as the `loading` state is no more managed, therefore I was wondering if that has been done on purpose or not

Reply:
yeah sure

Reply:
Done